main.js

```diff
@@ -20,6 +24,24 @@ define(function (require, exports, module) {
 	PreferencesManager.setViewState("afterFirstLaunch", false);
 	params.remove("skipSampleProjectLoad");
 
+	//This listener listens for commands from brambleproxy relating to buttons
+	function _listener(event) {
```

Comment:
Rename this function to something more indicative of why it's here. `_buttonListener` maybe?